.github/workflows/build-and-test.yml

```diff
@@ -213,6 +213,7 @@ jobs:
           git diff --exit-code '.github/ISSUE_TEMPLATE' || (echo 'Dropdowns in issue templates are out of date, please run "make generate-gh-issue-templates" and commit the changes in this PR.' && exit 1)
   unittest-matrix:
     strategy:
+      fail-fast: false
```

Comment:
For easy way to see if there are any unit test failures in all unit tests not only a single failed package